dryville_story.md

```diff
@@ -15,3 +15,17 @@ So let's say you made the big move. You and some friends have found your (desola
 
 In your free time go ahead and start digging a hole in the ground for a [well](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-wells). If you dig deep enough you might hit the [water table](https://www.usgs.gov/special-topic/water-science-school/science/aquifers-and-groundwater), where there will be standing water. You'll be able to lower a bucket down to get the available [groundwater](https://www.usgs.gov/special-topic/water-science-school/science/groundwater-what-groundwater). Things are starting out great -- you've already started using surface water (the creek and pond), ground water (your well), and you've even created your own water-storage system (the reservoir). So you now have a reliable source for your water needs. Time to relax? Not yet.
+
+## Getting Water to Your Homes
+
+Since you appointed yourself mayor Horace of Dryville, you naturally chose the best spot for your home — on top of the hill. Great view, but it sure is a pain lugging pails of water (at [over 8 pounds a gallon](https://www.usgs.gov/special-topics/water-science-school/science/water-density), your 100 gallons of water per day gets heavy in a hurry!) from the creek up the hill all day long. And now all your best buddies and neighbors are beginning to move in. Many of them are building on hills, too, so what you need now is a "water-supply distribution system" to get water to everyone's homes. The way to do this is to lay a system of pipes (which you make from clay from the creek bed) from the creek to each house.
+
+The problem is, the houses are higher than the creek, and water does not flow uphill. Water does flow downhill, so you build a big water storage tank on a hill (hopefully at the highest point in town) and establish a water brigade to fill it full of water. You can run pipes from the storage tank down to everyone's home. The pipes go right to your faucet and you use gravity (it's free!) to get the water delivered. Works great -- just like it did in the big city you came from.
+
+But you still have the problem of keeping the storage tank full. Lugging water up to it is no better than lugging it up to your house; so you need to find a way to pump the water from the creek uphill into the storage tank. Since Dryville doesn't have any industry to produce products to sell to the outside world to bring in money to buy things with yet, you need to build your own pump. And you can't use electricity since you haven't built a power plant yet (but you will). Can you think of a source of energy that can run your pump? A windmill could do the job, but in the desert there isn't always wind. Your creek! The water in your creek is flowing nicely -- so you build a paddle wheel in the river. The paddle spins in the creek's current and turns a rod that runs your homemade pump. Voila! You've got water being pumped uphill into your storage tank where gravity lets it flow to your homes.
+
+##Dryville's First Water Works
```

Comment:
I am not sure whether this will render correctly as a header without a space after the `##`, but in any case, it is easier to read if you include a space, consistent with how you've done it elsewhere. Could you please add a space?